frontend/App.tsx

```diff
@@ -1,10 +1,15 @@
 import { StatusBar } from 'expo-status-bar';
 import { StyleSheet, Text, View } from 'react-native';
-
+import useFetch from './utils/useFetch';
+import data from "./api/endpoint.json"
 export default function App() {
+  const [val, callUrl] = useFetch({url:data.baseApiUrl+data.getRegistrationForm.url,method:data.getRegistrationForm.method[0]});
+  function newCall(){
+    callUrl({})
+  }
   return (
     <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
+      <Text>{val}</Text>
       <StatusBar style="auto" />
     </View>
   );
```

Comment:
It's good to test our code, but remember: after that we should remove useFetch from this file.

Reply:
will do chief!